sections/testingandquality/3-parts-in-name.md

```diff
@@ -1,20 +1,20 @@
-# Include 3 parts in each test name

```

Comment:
Hey, thanks for the contribution! I see you mistakenly changed the English version instead of the Korean one